template/.github/ISSUE_TEMPLATE/publish_component.md

```diff
@@ -9,47 +9,62 @@ labels: 'Status: Review Needed, not-reviewed'
 
 ## Specific steps required to publish this component.
 
-All of the below outlined steps must be completed.
+1. __Design & Planning__
```

Comment:
It's not in this PR, but please checkout and update line 3, there are some typos. 

```
about: Create a ticket to drive compliance before publishing a a new componet to Auro Design System
```